collections/modis.yaml

```diff
@@ -194,7 +200,8 @@ Summaries:
           - 500
           - 500
         unit: m
-    - name: B07
+    - description: SWIR22 (2105 - 2155 nm)
+      name: B07
```

Comment:
Minor comment: Maybe update for all 3 collections the last modified entry at the bottom of each file.

Reply:
Thanks for the reminder. Updated.